app/recurring_job.go

```diff
@@ -360,6 +360,12 @@ func (job *Job) run() (err error) {
 		job.logger.Infof("Volume %v is in state %v", volumeName, volume.State)
 	}
 
+	if job.task == longhorn.RecurringJobTypeSnapshot || job.task == longhorn.RecurringJobTypeBackup {
```

Comment:
NIT: Add a simple comment explaining why

Reply:
@mantissahz It seems we can get this to 1.3.3 if this can get merged.

Reply:
@shuo-wu Added the simple comment to explain why only these two types.

@innobead yeah, let's try to do it.